### PR TITLE
Unlikely

### PR DESCRIPTION
**Pull request by Damian2809**
# New Port for Unlikely

## Game Information
- **Title**: Unlikely
- **URL**: https://nachogames.itch.io/unlikely

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/unlikely/cover.png
Comment (JanTrueno) [inline ports/unlikely/cover.png:1]: Can you get a higher quality version of this cover? And please compress it as well.

File: ports/unlikely/README.md
Comment (JanTrueno) [inline ports/unlikely/README.md:1]: Everything else?

File: ports/unlikely/screenshot.png
Comment (JanTrueno) [inline ports/unlikely/screenshot.png:1]: Compress the screenshot as well please.

**Review by JanTrueno — CHANGES_REQUESTED** [review]

File: ports/unlikely/README.md
Comment (Damian2809) [reply ports/unlikely/README.md:1]: what are you missing? i just used the basic readme.md

File: ports/unlikely/screenshot.png
Comment (JanTrueno) [reply ports/unlikely/screenshot.png:1]: I sent some revised images on discord :)

File: ports/unlikely/README.md
Comment (JanTrueno) [reply ports/unlikely/README.md:1]: "|Everything else|Left Mouse |"
This is what I mean ^^ Just tell the users what controls the left mouse click.

**Review by Cebion — APPROVED** [review]

**Review by JanTrueno — APPROVED** [review]

**Pull request merged**